DHIS2/post_clone_scripts/file_garbage_remover.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+
+import subprocess
+import sys
+import argparse
+
+
+LOCAL_SQL_FILE = "/tmp/find_orphan_files.sql"
+LOCAL_LIST_FILE = "/tmp/list_of_files_to_be_removed.txt"
+REMOTE_LIST_FILE = "/tmp/list_of_files_to_be_removed.txt"
+FILE_BASE_PATH = "/DHIS2_home/files"
+
+
+SQL_QUERY = """
+    SELECT storagekey
+    FROM fileresource fr
+    WHERE NOT EXISTS (
+        SELECT 1 FROM document d WHERE d.fileresource = fr.fileresourceid
+    )
+    AND fr.uid NOT IN (
+        SELECT url FROM document
+    )
+    AND fr.domain = 'DOCUMENT';
+"""
+
+
+def run(command, check=True, capture=False):
+    """Run a shell command."""
+    print(f"$ {' '.join(command)}")
+    return subprocess.run(command, check=check, capture_output=capture, text=True)
+
+
+def find_container_id(name_pattern):
+    """Find the container ID based on a name substring."""
+    result = run(["docker", "ps", "--format", "{{.ID}}\t{{.Names}}"], capture=True)
+    for line in result.stdout.strip().splitlines():
+        cid, name = line.split("\t")
+        if name_pattern in name:
+            return cid
+    return None
+
+
+def slugify(instance_name):
+    """Convert instance name to container name slug."""
+    return "d2-docker-" + instance_name.replace("/", "-").replace(":", "-").replace(".", "-")
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Remove orphaned DHIS2 file resources from Core container.")
+    parser.add_argument("-i", "--instance", required=True, help="d2-docker instance name (e.g. /usr/bin/python3 file_garbage_remover.py -i docker.eyeseetea.com/widpit/dhis2-data:2.42-widp-preprod-cont-indiv)")
+    args = parser.parse_args()
+
+    instance_name = args.instance
+    container_slug = slugify(instance_name)
+    db_container_match = container_slug + "-db-1"
+    core_container_match = container_slug + "-core-1"
```

Comment:
Deleted db_containter_match as it is not needed
Moved the replace to where the match string is constructed.
```suggestion
    core_container_match = container_slug.replace("dhis2-data-", "") + "-core-1"
```